asv/plugin_manager.py

```diff
@@ -42,9 +40,7 @@ def import_plugin(self, name):
             sys.path.insert(0, ".")
             name = name[1:]
         try:
-            mod = __import__(name, {}, {}, [], level=0)
-            self.init_plugin(mod)
-            self._plugins.append(mod)
+            self.load_plugins(name)
```

Comment:
This also fixes what I noticed in #1283, that local plugins are not actually loaded correctly. Now they are.